arviz/data/io_cmdstan.py

```diff
@@ -205,17 +205,16 @@ def sample_stats_to_xarray(self):
         dtypes = {"divergent__": bool, "n_leapfrog__": np.int64, "treedepth__": np.int64}
 
         sampler_params, sampler_params_warmup = self.sample_stats
-
+        rename_dict = {
+            "divergent__": "diverging",
+            "n_leapfrog__": "n_steps",
+            "treedepth__": "tree_depth",
+        }
         for j, s_params in enumerate(sampler_params):
-            rename_dict = {}
             for key in s_params:
-                key_, *end = key.split(".")
-                name = re.sub("__$", "", key_)
```

Comment:
Not sure about the split, @ahartikainen will know, but removing the final `__` is still necessary, there are some variables which are not renamed and should have them removed, `lp` for example

Reply:
Split is needed for nD variables

    theta.1.2

Reply:
No wait, I need to reread the code, it might be irrelevant in this step.

Reply:
> Not sure about the split, @ahartikainen will know, but removing the final `__` is still necessary, there are some variables which are not renamed and should have them removed, `lp` for example

In that case, all the stan variables in the [comment](https://github.com/arviz-devs/arviz/pull/1063#issuecomment-589771200) need to be mapped to unified name. To do that, a separate dictionary for renaming (other then already mentioned `rename_dict`) need to be made.

Reply:
Ok, I think that split is not necessary here.

I would use the rename dict for variables which we want to be mapped and then I would leave the rest of the variables as is (just removing `__` like we currently do)

Reply:
> I would leave the rest of the variables as is (just removing `__` like we currently do)

This assumes that there might be new diagnostics in future.

Reply:
Okay. 
Then I am initializing `rename_dict` before the outer for loop.